src/pages/resource/DescriptionPanel.tsx

```diff
@@ -18,6 +16,23 @@ import { defaultLanguage, languages } from '../../translations/i18n';
 import { ResourceFormTabs } from '../../types/resource.types';
 import publications from '../../utils/testfiles/projects_random_generated.json';
 import FormikDatePicker from './FormikDatePicker';
+import styled from 'styled-components';
+import Box from '../../components/Box';
+
+const MultipleFieldWrapper = styled.div`
+  display: flex;
+`;
+
+const StyledFieldWrapper = styled.div`
+  padding: 1rem;
+  flex: 1 0 40%;
+`;
```

Comment:
Prøvde å få det til å virke med å extende `Field` i stedet for `div`, for å unngå en ekstra "unødvendig" tag, men det ble egentlig bare mer rot